### DISCUSSION TITLE
Keybind to compile with XeLaTeX

### DISCUSSION THREAD

**Discussion by brunood1** [James-Yu/LaTeX-Workshop#3562] [Q&A]
How can I set a custom keybind to compile with XeLaTeX, I use XeLaTeX in one of my documents and a lot of the times because of muscle memory I end up compiling with pdfLaTeX and so I want to set a XeLaTeX keybind for that.

**Comment by jlelong** [comment]
It is not possible to bind a key sequence to a specific recipe. Yet, you can customize which recipe to use per default using [`latex-workshop.latex.recipe.default`](https://github.com/James-Yu/LaTeX-Workshop/wiki/Compile#latex-workshoplatexrecipedefault).
